Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM eclipse-temurin:21-jre-jammy AS builder
+FROM --platform=$BUILDPLATFORM eclipse-temurin:21-jdk-jammy AS builder
```

Comment:
So there has been a Slack thread on this but there was no real conclusion as to why this change is needed.

Either way this is just for building it and not the actual running i.e. still using JRE for running.